actor/rage.js

```diff
@@ -158,3 +185,15 @@ if (chatMsg !== '') {
     };
 	ChatMessage.create(chatData, {});
 }
+© 2020 GitHub, Inc.
+Terms
+Privacy
+Security
+Status
+Help
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
```

Comment:
And this too.

Reply:
Requested changes made, thanks for that.